lib/services/authentication-service.ts

```diff
@@ -60,7 +60,7 @@ export class AuthenticationService implements IAuthenticationService {
 			if (!options.openAction) {
 				this.$opener.open(loginUrl);
 			} else {
-				options.openAction(loginUrl);
+				options.openAction(loginUrl).catch(reject);
```

Comment:
We've discussed it with Tsvetie and we've agreed that openAction will not be a promise, so we do not need this catch. Just update the method definition and Public API please.